tools/.github-cla-signers

```diff
@@ -143,6 +143,7 @@ onitake
 orndorffgrant
 Oursin
 outscale-mdr
+cpaelzer
```

Comment:
WAIT WHAT NO WAY THIS IS YOUR FIRST PR

Reply:
Nope just wasn't pulled into our tools/.github-cla-signers.
First commit eeabe2ed7480dc653c9d65d218738a8ed5a21579